### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Keyboard: In high contrast mode, focus is not visible on "Results: untitled-1" tab

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/vscode-mssql#1235]
**Environment:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.17763 Build 17763)

**MAS ID Violated:** MAS 2.4.7, MAS 1.4.3

**Repro Steps:**
1. Open Visual Studio Code in your system
2. Navigate to "Extensions" and install "SQl Server (mssql)" extension
3. Press F1 and select MS SqL : Connect and connect to any sql server
4. "Write  code (SELECT *
FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""  "
5. Change the color theme to high contrast(Files-->Preferences-->Color Theme-->High Contrast)
6. Tab till "Results: untitled-1" tab and verify user is able to see the focus visibility in high contrast mode or not

**Actual Result:**
In high contrast mode, focus is not visible on "Results: untitled-1" tab

**Expected Result:**
In high contrast mode, focus should be visible on "Results: untitled-1" tab

**User Impact:**
Keyboard users and low vision users will finds difficult while working if they are unable to see the focus visibility on "Results: untitled-1" tab

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachments:**
[Bug_1235_focus visibility at tab item in high contrast mode.pptx](https://github.com/Microsoft/vscode-mssql/files/3111604/Bug_1235_focus.visibility.at.tab.item.in.high.contrast.mode.pptx)

**Comment by alanrenmsft** [comment]
@InduPriya1805 I think this is an upstream issue (vscode), when a tab is current the active tab, the focus rectangle will not be shown.

**Comment by alanrenmsft** [comment]
testers, please open an issue for vscode. as per Melissa’s advice, I am closing this issue.

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by Roopesh-T** [comment]
#closed;
